### PR TITLE
save payer_id

### PR DESCRIPTION
**Pull request by daliwei**
@pierre Please review. The purpose of this PR is to save payer_id in the complete_auth/purchase response.

### CODE REVIEW

**Comment by daliwei** [comment]
@pierre I have updated the codes, which relies on this [PR](https://github.com/killbill/killbill-plugin-framework-ruby/pull/56). Please take a look.

**Comment by pierre** [comment]
👍 Awesome, much simpler!

File: lib/paypal_express/api.rb [hunk 1]
```diff
@@ -425,7 +425,7 @@ def authorize_or_purchase_payment(kb_account_id, kb_payment_id, kb_payment_trans
           end
 
           properties = merge_properties(properties, options)
-          dispatch_to_gateways(api_call_type, kb_account_id, kb_payment_id, kb_payment_transaction_id, kb_payment_method_id, amount, currency, properties, context, gateway_call_proc)
+          dispatch_to_gateways(api_call_type, kb_account_id, kb_payment_id, kb_payment_transaction_id, kb_payment_method_id, amount, currency, properties, context, gateway_call_proc, nil, {:payer_id => options[:payer_id]})
```
Comment (pierre) [inline lib/paypal_express/api.rb]: Should we check if `payer_id` is `nil`? Otherwise, we might overwrite a real value returned by PayPal?

Comment (daliwei) [inline lib/paypal_express/api.rb]: Looks like `payer_id` cannot be `nil`. Either it was passed in or returned from paypal.

Comment (pierre) [inline lib/paypal_express/api.rb]: Correct - thanks for checking!

**Pull request merged**